### DISCUSSION TITLE
Disable local login when using oauth

### DISCUSSION THREAD

**Discussion by thomaskenne** [statamic/cms#4599] [Q&A] [answered]
Hi. 

Is it possible to disable the "or Login with email" when using oauth as login mechanism?

**Comment by jasonvarga** [comment, marked as answer]
~Not at the moment but it's a fair request.~

Edit: It's possible in 3.2.22

**Reply by duncanmcclean** [reply]
For reference (as it won't show in here), created a pull request for this: #4625

**Comment by ARCHES-Academy** [comment]
We had a similar use case for our site, so we solved it for our site by changing the login links from pointing to `/cp` to `/oauth/{provider}?redirect=/cp`. The login form still existed, but we didn't use it. It just went directly to google's form (or whichever oauth provider you're using).

You could also just build a new login form with only the options you'd like represented, that way you have complete control. You can read more about that in the docs [here](https://statamic.dev/tags/user-login_form): https://statamic.dev/tags/user-login_form
